### DISCUSSION TITLE
Optimal Setting of Qwen Lora training for characters and style?

### DISCUSSION THREAD

**Discussion by krigeta** [kohya-ss/musubi-tuner#504] [Q&A]
My issue won't be hard for you to understand. All I am looking for is to train a Qwen LoRA (not the Qwen Edit). I need to train some character LoRAs or a single multi-character LoRA, if possible, and some style LoRAs.

The characters I am training are limited when it comes to the number of images, as they are original characters (edited: some characters like Goku, etc.).

So I am trying to train a LoRA of them first, using game screenshots. I have around 20-50 images per character, but I don't know what settings I should use. A learning rate of 5e-5 seems low, and I guess it will learn very slowly. I am GPU-poor, so I am using cloud services.
My goal is to know what settings would be best for me to train a single/multi-character LoRA so it won't take too much time, won't overfit, but will work better than expected, even for extreme poses.
